specification/sbi_cove.adoc

```diff
@@ -109,6 +122,15 @@ The host is responsible for the following memory management functions:
 . Mapping zero-page confidential pages to the TVM regions.
 . Mapping non-confidential pages TVM-defined regions for shared-pages / MMIO.
 
+[NOTE]
+====
+The division into confidential and non-confidential memory might be done statically or dynamically. 
+CoVE implementations that support only static partitioning of confidential and 
+non-confidential memory (for example, high-assurance systems that must be formally verified, see <<appendix_d>>) 
```

Comment:
```suggestion
non-confidential memory (for example, see <<appendix_d>>) 
```

Reply:
systems with dynamic confidential memory may also be formally verified and high assurance so I removed that qualification from this example reference.